src/base/utils/fs.cpp

```diff
@@ -377,137 +360,17 @@ QString Utils::Fs::expandPathAbs(const QString& path)
 
 QString Utils::Fs::QDesktopServicesDataLocation()
```

Comment:
Why do you leave these ugly wrappers?
Why shouldn't we use Environment utils directly?